doc/nrf/app_dev/device_guides/hpf/power_management.rst

```diff
@@ -0,0 +1,209 @@
+.. _hpf_power_management:
+
+Power management
+################
+
+The following page outlines the approaches and mechanisms employed to ensure efficient power management in High-Performance Framework (HPF) systems, focusing on resource allocation, system states, and interaction between different components.
+
+Power Management in HPF should be considered from the Host side and the HPF firmware side.
+
+Host side
+*********
+
+The Host is responsible for managing power state of the following resources:
+
+* VPR core with HPF firmware running on it
+* RAM
+
+To achieve optimal power state, you must use the `Device Runtime Power Management`_ and `System Power Off`_.
+
+See an example flow of the Host power management:
+
+  .. uml::
```

Comment:
Are we implementing it like this in mSPI?
If not, then better to use other name of driver.